frontend/src/locales/en.yml

```diff
@@ -78,11 +84,11 @@ settings:
     plan: (Upgrade to Team plan to enable role management)
   workspace:
     url-section: SQL Console URL
-    url-tip: "Tip: Use {schema} as the placeholder for the actual
-          database name"
-    tip: If your team use a separate SQL console such as phpMyAdmin, you can
-        provide its URL pattern here. Once provided, Bytebase will surface the
-        console link on the relevant database and table UI.
+    url-tip: 'Tip: Use %s as the placeholder for the actual database name'
```

Comment:
Seems the change for remove `%s` is revert

Reply:
My bad. Done